cmake/IgnUtils.cmake

```diff
@@ -936,7 +936,7 @@ function(ign_create_core_library)
   # Create the target for the core library, and configure it to be installed
   _ign_add_library_or_component(
     LIB_NAME ${PROJECT_LIBRARY_TARGET_NAME}
-    INCLUDE_DIR "ignition/${IGN_DESIGNATION_LOWER}"
+    INCLUDE_DIR "${PROJECT_INCLUDE_DIR}"
     EXPORT_BASE IGNITION_${IGN_DESIGNATION_UPPER}
```

Comment:
> I noticed that `IGN_DESIGNATION_LOWER` is being replaced here, instead of `IGN_DESIGNATION`. It looks like `IGN_DESIGNATION_LOWER` is the lowercase version of `IGN_DESIGNATION`:
> 
> https://github.com/ignitionrobotics/ign-cmake/blob/4f40889950da9ab57391d8fa5465461a2837f006/cmake/IgnConfigureProject.cmake#L76
> 
> Should we create and use something like `PROJECT_INCLUDE_DIR_LOWER` here for consistency?

`IGN_DESIGNATION` is a special variable that we use in lots of different places with different cases, which is why the `IGN_DESIGNATION_LOWER` variable was created. I don't think we need different case versions of `PROJECT_INCLUDE_DIR` because we just use it in one place

> Also, on a side note, I noticed that `IGN_DESIGNATION_UPPER` is being used in the line below the changes here. While this isn't relate to the include directories work of this PR, I am guessing this is something we'll have to update in another PR if we want to increase flexibility for things like sdformat.

this is the `EXPORT_BASE` issue, which is mentioned in the 3rd checkbox in https://github.com/ignitionrobotics/sdformat/issues/181#issuecomment-962116041. I think we can address that independently